sources/procfs.go

```diff
@@ -125,68 +125,68 @@ func newLustreProcMetric(filename string, promName string, source string, path s
+			{"job_stats", "ost_job_quotactl_total", quotactlHelp},
+			{"kbytesavail", "ost_kilobytes_available", "Number of kilobytes readily available in the pool"},
+			{"kbytesfree", "ost_kilobytes_free", "Number of kilobytes allocated to the pool"},
+			{"kbytestotal", "ost_kilobytes_capacity", "Capacity of the pool in kilobytes"},
+			{"lfsck_speed_limit", "ost_lfsck_speed_limit", "Maximum operations per second LFSCK (Lustre filesystem verification) can run"},
+			{"num_exports", "ost_exports_total", "Total number of times the pool has been exported"},
+			{"precreate_batch", "ost_precreate_batch", "Maximum number of objects that can be included in a single transaction"},
+			{"recovery_time_hard", "ost_recovery_time_hard_seconds", "Maximum timeout 'recover_time_soft' can increment to for a single server"},
+			{"recovery_time_soft", "ost_recovery_time_soft_seconds", "Duration in seconds for a client to attempt to reconnect after a crash (automatically incremented if servers are still in an error state)"},
+			{"soft_sync_limit", "ost_soft_sync_limit", "Number of RPCs necessary before triggering a sync"},
+			{"stats", "ost_read_samples_total", readSamplesHelp},
+			{"stats", "ost_read_minimum_size_bytes", readMinimumHelp},
+			{"stats", "ost_read_maximum_size_bytes", readMaximumHelp},
+			{"stats", "ost_read_bytes_total", readTotalHelp},
+			{"stats", "ost_write_samples_total", writeSamplesHelp},
+			{"stats", "ost_write_minimum_size_bytes", writeMinimumHelp},
+			{"stats", "ost_write_maximum_size_bytes", writeMaximumHelp},
+			{"stats", "ost_write_bytes_total", writeTotalHelp},
+			{"sync_journal", "ost_sync_journal_enabled", "Binary indicator as to whether or not the journal is set for asynchronous commits"},
+			{"tot_dirty", "ost_exports_dirty_total", "Total number of exports that have been marked dirty"},
+			{"tot_granted", "ost_exports_granted_total", "Total number of exports that have been marked granted"},
+			{"tot_pending", "ost_exports_pending_total", "Total number of exports that have been marked pending"},
 		},
 		"ldlm/namespaces/filter-*": {
-			{"lock_count", "lock_count", "Number of locks"},
-			{"lock_timeouts", "lock_timeouts", "Number of lock timeouts"},
-			{"contended_locks", "contended_locks", "Number of contended locks"},
-			{"contention_seconds", "contention_seconds", "Time in seconds during which locks were contended"},
-			{"pool/granted", "granted", "Number of granted locks"},
-			{"pool/grant_rate", "grant_rate", "Lock grant rate"},
-			{"pool/cancel_rate", "cancel_rate", "Lock cancel rate"},
-			{"pool/grant_speed", "grant_speed", "Lock grant speed"},
+			{"lock_count", "ost_lock_count_total", "Number of locks"},
+			{"lock_timeouts", "ost_lock_timeout_total", "Number of lock timeouts"},
+			{"contended_locks", "ost_lock_contended_total", "Number of contended locks"},
+			{"contention_seconds", "ost_lock_contention_seconds", "Time in seconds during which locks were contended"},
```

Comment:
Is this a counter? As in, do we need to append `_total`? Don't know enough about `contention_seconds` off-hand to determine one way or another with certainty.